packages/account/src/Components/form-field-info/form-field-info.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Popover } from '@deriv/components';
+import { TPopoverProps } from '@deriv/components/src/components/types';
+import { isMobile } from '@deriv/shared';
+
+/**
+ * A component that renders a popover with an info icon.
+ *
+ * @param {TPopoverProps} props - Props for the popover component.
+ * @returns React.ReactElement - A React component.
+ */
+export const FormFieldInfo = (props: Omit<TPopoverProps, 'alignment'>) => {
+    const [is_open, setIsOpen] = React.useState(false);
+    const ref = React.useRef<HTMLDivElement>(null);
+
+    React.useEffect(() => {
+        // Close popover when clicking outside of it
+        const handleClickOutside = (event: MouseEvent) => {
```

Comment:
clarification: Isn't this handled by popover component

Reply:
No @amina-deriv, it's not. It should've been handled within the popover component yet we don't have such implementation in there